Changelog.md

```diff
@@ -175,337 +175,337 @@
-    * [Pull request 373](https://bitbucket.org/osrf/sdformat/pull-request/373)
+    * [BitBucket pull request 373](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/373)
 
 ### SDFormat 4.3.2 (2017-07-19)
 
 1. Add documentation for `Element::GetFirstElement()` and `Element::GetNextElement()`
-    * [Pull request 341](https://bitbucket.org/osrf/sdformat/pull-request/341)
+    * [BitBucket pull request 341](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/341)
 
 1. Fix parser to read plugin child elements within an `<include>`
-    * [Pull request 350](https://bitbucket.org/osrf/sdformat/pull-request/350)
+    * [BitBucket pull request 350](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/350)
 
 ### SDFormat 4.3.1 (2017-03-24)
 
 1. Fix segmentation Fault in `sdf::getBestSupportedModelVersion`
-    * [Pull request 327](https://bitbucket.org/osrf/sdformat/pull-requests/327)
-    * [Issue 152](https://bitbucket.org/osrf/sdformat/issues/152)
+    * [BitBucket pull request 327](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-requests/327)
+    * [Issue 152](https://github.com/osrf/sdformat/issues/152)
 
 ### SDFormat 4.3.0 (2017-03-20)
 
 1. Choosing models with more recent sdf version with `<include>` tag
-    * [Pull request 291](https://bitbucket.org/osrf/sdformat/pull-request/291)
-    * [Issue 123](https://bitbucket.org/osrf/sdformat/issues/123)
+    * [BitBucket pull request 291](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/291)
+    * [Issue 123](https://github.com/osrf/sdformat/issues/123)
 
 1. Added `<category_bitmask>` to 1.6 surface contact parameters
-    * [Pull request 318](https://bitbucket.org/osrf/sdformat/pull-request/318)
+    * [BitBucket pull request 318](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/318)
 
 1. Support light insertion in state
-    * [Pull request 325](https://bitbucket.org/osrf/sdformat/pull-request/325)
+    * [BitBucket pull request 325](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/325)
 
 1. Case insensitive boolean strings
-    * [Pull request 322](https://bitbucket.org/osrf/sdformat/pull-request/322)
+    * [BitBucket pull request 322](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/322)
 
 1. Enable coverage testing
-    * [Pull request 317](https://bitbucket.org/osrf/sdformat/pull-request/317)
+    * [BitBucket pull request 317](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/317)
 
 1. Add `friction_model` parameter to ode solver
-    * [Pull request 294](https://bitbucket.org/osrf/sdformat/pull-request/294)
+    * [BitBucket pull request 294](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/294)
     * [Gazebo pull request 1522](https://bitbucket.org/osrf/gazebo/pull-request/1522)
```

Comment:
Did you leave this out on purpose?

Reply:
Yeah it was on purpose, but then I realized it wasn't worth it coming back just to fix those, so the latest pass addresses that as well: 16dcc94 .

The link itself will only become valid once `osrf/gazebo` has been fully migrated.